InstanceAdminGaxTest.getInstanceConfigTest is flaky

https://github.com/googleapis/java-spanner/pull/81/checks?check_run_id=764355977

```
Jun 12, 2020 5:52:50 AM io.grpc.internal.SerializingExecutor run
SEVERE: Exception while executing runnable io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1HalfClosed@6ee029be
java.util.NoSuchElementException
	at java.util.LinkedList.removeFirst(LinkedList.java:270)
	at java.util.LinkedList.remove(LinkedList.java:685)
	at com.google.cloud.spanner.admin.instance.v1.MockInstanceAdminImpl.getInstanceConfig(MockInstanceAdminImpl.java:96)
	at com.google.spanner.admin.instance.v1.InstanceAdminGrpc$MethodHandlers.invoke(InstanceAdminGrpc.java:1638)
	at io.grpc.stub.ServerCalls$UnaryServerCallHandler$UnaryServerCallListener.onHalfClose(ServerCalls.java:172)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.halfClosed(ServerCallImpl.java:331)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1HalfClosed.runInContext(ServerImpl.java:818)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

[ERROR] Tests run: 42, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 4.735 s <<< FAILURE! - in com.google.cloud.spanner.InstanceAdminGaxTest
[ERROR] com.google.cloud.spanner.InstanceAdminGaxTest.getInstanceConfigTest[exception at call = 0, exception type = DELAYED]  Time elapsed: 0.089 s  <<< ERROR!
com.google.cloud.spanner.SpannerException: UNKNOWN: com.google.api.gax.rpc.UnknownException: io.grpc.StatusRuntimeException: UNKNOWN
	at com.google.cloud.spanner.InstanceAdminGaxTest.getInstanceConfigTest(InstanceAdminGaxTest.java:373)
Caused by: java.util.concurrent.ExecutionException: com.google.api.gax.rpc.UnknownException: io.grpc.StatusRuntimeException: UNKNOWN
	at com.google.cloud.spanner.InstanceAdminGaxTest.getInstanceConfigTest(InstanceAdminGaxTest.java:373)
Caused by: com.google.api.gax.rpc.UnknownException: io.grpc.StatusRuntimeException: UNKNOWN
Caused by: io.grpc.StatusRuntimeException: UNKNOWN
```

--- a/google-cloud-spanner/src/test/java/com/google/cloud/spanner/InstanceAdminGaxTest.java
+++ b/google-cloud-spanner/src/test/java/com/google/cloud/spanner/InstanceAdminGaxTest.java
@@ -224,10 +224,10 @@ public class InstanceAdminGaxTest {
     RetrySettings retrySettingsWithLowTimeout =
         RetrySettings.newBuilder()
             .setInitialRetryDelay(Duration.ofMillis(1L))
-            .setMaxRetryDelay(Duration.ofMillis(1L))
+            .setMaxRetryDelay(Duration.ofMillis(1000L))
             .setInitialRpcTimeout(Duration.ofMillis(20L))
             .setMaxRpcTimeout(Duration.ofMillis(200L))
-            .setRetryDelayMultiplier(1.3)
+            .setRetryDelayMultiplier(1000.0d)
             .setMaxAttempts(10)
             .setTotalTimeout(Duration.ofMillis(200L))
             .build();

